### PR TITLE
Views (dam)

### PR DESCRIPTION
**Pull request by Polymorphe57**

### CODE REVIEW

File: addons/web/static/tests/views/view_tests.js [hunk 1]
```diff
@@ -778,6 +736,26 @@ QUnit.module("Views", (hooks) => {
...
+        assert.expect(2);
+        try {
+            await makeView({ serverData, resModel: "animal", type: "toy", arch: "<toy/>" });
+        } catch (error) {
+            assert.step(error.message);
+        }
+        assert.verifySteps([`"arch" and "fields" props must be given together`]);
+    });
+
+    QUnit.test("'fields' cannot be passed as prop alone", async function (assert) {
```
Comment (Polymorphe57) [inline addons/web/static/tests/views/view_tests.js:803]: add similar assertions for search related things

File: addons/project/static/src/js/burndown_chart/burndown_chart_view.js [hunk 2]
```diff
@@ -6,8 +6,8 @@ import { registry } from "@web/core/registry";
 
 const viewRegistry = registry.category("views");
 
-class BurndownChartView extends GraphView {};
-BurndownChartView.components = { BurndownChartRenderer };
+class BurndownChartView extends GraphView {}
+BurndownChartView.components = { ...GraphView.components, BurndownChartRenderer };
```
Comment (Polymorphe57) [inline addons/project/static/src/js/burndown_chart/burndown_chart_view.js]: was working... check why

Comment (Polymorphe57) [reply addons/project/static/src/js/burndown_chart/burndown_chart_view.js]: t t-component :-)

**Comment by ged-odoo** [comment]
robodoo r+

**Comment by ged-odoo** [comment]
robodoo rebase-merge

**Pull request closed without merging**